CRISPResso2/CRISPRessoPlot.py

```diff
@@ -382,6 +453,8 @@ def plot_frequency_deletions_insertions(
     xmax_ins,
     xmax_mut,
     save_also_png=False,
+    custom_colors=None,
```

Comment:
It should be `None`, as handled by `get_color_lookup()` (lines 81-96)

Reply:
Then do we need to default the `custom_colors` to `None` in all of the plot functions? If I remember right I think `None` was passed in for all of the other functions.

Reply:
okay actually, maybe @Snicker7 knows a little more about `custom_config['colors']` but, it looks like we always load in at least default colors, and then for most of the plot function calls it's passed as a positional argument (as @Colelyman said). SO could we just remove all default `None` values?

Reply:
As far as I can tell we could move this to positional and remove the default value (because it works for the other plot functions)... but I haven't tested it.